interface/reports/patient_list_creation.php

```diff
@@ -97,8 +97,8 @@ function add_date($givendate, $day = 0, $mth = 0, $yr = 0)
         var global_date_format = '%Y-%m-%d';                
         function Form_Validate() {
             var d = document.forms[0];       
-            FromDate = d.date_from.value;
-            ToDate = d.date_to.value;
+            FromDate = d.form_from_date.value;
```

Comment:
Thank you for standardizing this.  It is irritating to have to look up the variable in report after report (uh, which one is it here???).  The same applies to provider_id, providerid, doc_id, ProviderID, and providerID.  I am sure there are other criteria that should be standardized.  Be on lookout for "pubpid" being used as well.

Reply:
Thanks for this @aethelwulffe